mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -165,35 +169,73 @@ class MDocUpdater : MDocCommand
 
 	HashSet<string> forwardedTypes = new HashSet<string> ();
 
-	public static string droppedNamespace = string.Empty;
+	public static DroppedAssemblyInfo.List droppedAssemblyList = new DroppedAssemblyInfo.List ();
+	public static string currentDroppedNamespace = string.Empty;
```

Comment:
It can wait.

Overall, I have two issues with this patch:

1. It's *gigantic*, which makes reviewing it hard.
2. One of the reasons why it's gigantic is because of pointless whitespace changes (mentioned elsewhere, but not all whitespace changes were commented upon).

The whitespace changes need to die.

That aside, is there some way to perhaps split this up into smaller changes?